woocommerce/woocommerce-android
# Printing receipts instructions screen as dialog on tablets

Closes: #10649 


### Description

This PR modifies the printing receipts instructions fragment to display as a dialog on tablet devices, maintaining visibility of the underlying context and ensuring an optimal UI appearance. Conversely, on phones, the printing receipts instructions fragment should continue to operate in a full-screen mode.

### Testing instructions

1. Open order listing screen on tablet device
2. Select order item that's pending payment
3. Click on "Learn more about printing receipts with your device" text below `Collect Payment` button
4. Ensure you see a dialog with the instructions
5. Run through the same steps on mobile phone and ensure the instructions screen appears as full screen on phones.

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/1331230/e07dc54b-8cfa-4a92-9d37-88d2f023518e
